package.json

```diff
@@ -590,6 +599,18 @@
 					"command": "autolisp.insertFoldingRegion",
 					"group": "navigation@3"
 				}
+			],
+			"explorer/context": [
+				{
+					"when": "resourceLangId == autolisp && autolisp.hasProject",					
+					"command": "autolisp.addWorkspaceFile2Project",
+					"group": "navigation@1"
+				},
+				{
+					"when": "explorerResourceIsFolder && autolisp.hasProject",					
```

Comment:
This is, but I'll draw your attention Line 99 simply because it is where I recycled the previous localization that I am questioning should change to possibly pluralize the localized statement.